<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(255, 170, 255), stop:1 rgb(255, 170, 255))
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(251, 194, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 85, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(255, 85, 255);
font:bold 14px;
min-width: 10em;
padding: 6px;
</string>
    </property>
    <property name="title">
     <string>LogIn</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Username">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>79</y>
       <width>186</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>129</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="PushButton_Login">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>240</y>
       <width>186</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>LogIn</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>186</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>186</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(251, 194, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 85, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(255, 85, 255);
font: bold 14px;
min-width: 10em;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>LogIn to your Market Place</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
